src/github.com/getlantern/flashlight/config/flags.go

```diff
@@ -89,3 +100,14 @@ func (updated *Config) applyFlags() error {
 
 	return nil
 }
+
+func parseFrontFQDNs(frontFQDNs string) (map[string]string, error) {
+	fqdns := map[string]string{}
+	if err := yaml.Unmarshal([]byte(frontFQDNs), fqdns); err != nil {
+		return nil, err
+	}
+	if len(fqdns) == 0 {
```

Comment:
I like it.

Just for the record, there's one more weird edge case.  Let's say that I previously configured the app with some fqdns via the command line - this will now be captured in the YAML.  If you allowed an empty list from the command line, I could use the command line to override what was in the YAML.  If you don't, I can't.  I can't actually think of a legitimate use case for this, so I wouldn't worry about it, but it's a weird nuance about how the configuration system works that I'm not 100% comfortable with and I wanted to make you aware of it.